scripts/init.sh

```diff
@@ -1,3 +1,9 @@
+# Check if the script is being sourced
+if [[ "${BASH_SOURCE[0]}" == "${0}" ]]; then
+    echo "This script must be sourced. Run it with: source $0"
```

Comment:
Can we add warning may be in yellow color?

Reply:
updated